tests/pytests/integration/modules/test_ldap3.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+
+pytest_plugins = [
+    "tests.support.pytest.ldap",
+]
```

Comment:
```suggestion
from tests.support.pytest.ldap import openldap_minion_run
```
Import the fixture instead.